README.md

```diff
@@ -56,9 +57,22 @@ julia> Pkg.add("Trixi"); Pkg.add("Trixi2Vtk"); Pkg.add("Trixi2Img")
 ```
 Note that you can copy and paste all commands to the REPL *including* the leading
 `julia>` prompts - they will automatically be stripped away by Julia.
-Further details can be found in the [documentation](#documentation), including
-instructions on how to install Trixi such that you can modify or extend it with
-your own implementations.
+
+### For developers
+If you plan on editing Trixi itself, you can download Trixi locally and run it from
+within the cloned directory:
+```bash
+git clone git@github.com:trixi-framework/Trixi.jl.git
+cd Trixi.jl
+julia --project=. -e 'import Pkg; Pkg.instantiate()' # Install Trixi's dependencies
+julia -e 'import Pkg; Pkg.add("Trixi2Vtk"); Pkg.add("Trixi2Img")' # Install postprocessing tools
+```
+If you installed Trixi this way, you always have to start Julia with the `--project`
+flag set to your local Trixi clone, e.g.,
+```bash
+julia --project=.
+```
+Further details can be found in the [documentation](#documentation).
```

Comment:
What about the possibility to
```julia
Pkg.add("Trixi")
```
at first (as a user) and
```julia
Pkg.dev("Trixi") # clones the repository, usually to ~/.julia/dev/Trixi on Linux
```
afterwards (as a developer)? At least that's the approach I prefer. Then, one doesn't have to start Julia with any special project flags or `instatiate` stuff.

Reply:
To elaborate a bit: What's described in the current version is the approach for projects (in the sense of scientific projects like our repro repo). In contrast, what I've described above is what I consider the approach for packages (in the sense of libraries). I like to think of Trixi in the second way - let's provide building blocks to run simulations in a library instead of developing a code that has to be compiled and modified from scratch for each paper.

Reply:
OK, let's discuss this at the meeting later. While I generally agree with what you've said here, I personally like the "manual" approach better, as it allows to easily switch between different local Trixi folders without getting confused about which version I am currently using. But maybe I'm still doing it wrong ;-)

Reply:
Fixed in aa551e4 as per discussion.